lib/client/mfa/prompt.go

```diff
@@ -31,33 +29,34 @@ import (
 	"github.com/gravitational/teleport"
 	"github.com/gravitational/teleport/api/client/proto"
 	"github.com/gravitational/teleport/api/observability/tracing"
-	"github.com/gravitational/teleport/api/utils/prompt"
 	wancli "github.com/gravitational/teleport/lib/auth/webauthncli"
 	wantypes "github.com/gravitational/teleport/lib/auth/webauthntypes"
-	"github.com/gravitational/teleport/lib/auth/webauthnwin"
 )
 
-// AdminMFAHintBeforePrompt is a hint used for MFA prompts for admin-level API requests.
-const AdminMFAHintBeforePrompt = "MFA is required for admin-level API request."
-
 var log = logrus.WithFields(logrus.Fields{
 	trace.Component: teleport.ComponentClient,
 })
 
 // Prompt is an MFA prompt.
 type Prompt struct {
-	// WebauthnLogin performs client-side Webauthn login.
-	WebauthnLogin func(ctx context.Context, origin string, assertion *wantypes.CredentialAssertion, prompt wancli.LoginPrompt, opts *wancli.LoginOpts) (*proto.MFAAuthenticateResponse, string, error)
+	// PromptMFA is an interchangeable MFA prompt handler using the common config options below.
+	PromptMFA
```

Comment:
Could we avoid the embed, or do we have clients that need access to specific methods? (Maybe expose just the webauthn one?)